src/Util.php

```diff
@@ -75,4 +75,71 @@ public static function getTimestamp() {
         $micro = sprintf("%06d", ($t - floor($t)) * 1000000);
         return date('Y-m-d\TH:i:s.' . $micro . 'O', $t);
     }
+
+     /*
```

Comment:
This way, this will not be treated as a docblock (it needs to start with `/**`).